runtime/parachains/src/builder.rs

```diff
@@ -221,8 +220,11 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 
 	/// Get the minimum number of validity votes in order for a backed candidate to be included.
 	#[cfg(feature = "runtime-benchmarks")]
-	pub(crate) fn fallback_min_validity_votes() -> u32 {
-		(Self::fallback_max_validators() / 2) + 1
+	pub(crate) fn fallback_min_backing_votes() -> u32 {
+		use crate::inclusion::minimum_backing_votes;
+		minimum_backing_votes(Self::fallback_max_validators_per_core() as _)
```

Comment:
Because the would require passing in `&self` and that I was not able to make to work with the benchmarking macro. :-S Anyhow, I'll see whether I can clean this up somehow.